lib/blueberry_rails/app_builder.rb

```diff
@@ -4,18 +4,19 @@ class AppBuilder < Rails::AppBuilder
     include BlueberryRails::ActionHelpers
 
     def readme
-      template 'README.md.erb', 'README.md'
+      copy_file 'README.md', 'README.md'
     end
 
-    def replace_gemfile
-      remove_file 'Gemfile'
+    def gitignore
+      copy_file 'gitignore_custom', '.gitignore'
+    end
+
+    def gemfile
       template 'Gemfile_custom.erb', 'Gemfile'
     end
 
-    def replace_secret_token
-      template 'secret_token.rb.erb',
-        'config/initializers/secret_token.rb',
-        force: true
+    def secret_token
+      template 'secret_token.rb.erb', 'config/initializers/secret_token.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.